sidekiq/manifest.json

```diff
@@ -0,0 +1,33 @@
+{
+  "display_name": "Sidekiq",
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "sidekiq",
+  "metric_prefix": "sidekiq.",
+  "metric_to_check": "sidekiq.jobs.count",
+  "creates_events": false,
+  "short_description": "Track metrics about your Sidekiq jobs, queues, and batches.",
+  "guid": "b4bc604c-73a5-4bd8-8dfe-3f80fc19976b",
+  "support": "core",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "Datadog-Sidekiq Integration",
+  "categories": [
+    "log collection"
```

Comment:
Since we also collect dogstatsd metrics, can we include the category for sidekiq metric types?

Reply:
Sure!